moveit_py/moveit/servo_client/teleop.py

```diff
@@ -52,13 +52,14 @@ class TeleopDevice(ABC, Node):
     This class expects both the `publish_command` and `record` methods to be overridden by inheriting classes.
     """
 
-    def __init__(self, node_name, device_name, device_config, ee_frame_name):
+    def __init__(self, node_name, device_name, device_config):
         super().__init__(node_name=node_name)
         self.device_name = device_name
         self.device_config = device_config
-        self.ee_frame_name = ee_frame_name
+        self.declare_parameters(namespace=node_name, parameters=[("ee_frame_name","panda_link8")])
```

Comment:
Is any of the issues remaining with the following where I can contribute a little ?

Reply:
Hi @Tejal-19 - this item remains open feel free to fix it in order to contribute to the pr.

Reply:
> This parameter should be assigned to the node within the launch file. In this case I think we want to read in this parameter from a `servo.yaml` config file ([example file](https://github.com/ros-planning/moveit2/blob/0f47f28d4e527362da421a51fe73a8abc1f716d1/moveit_ros/moveit_servo/config/panda_simulated_config.yaml#L45))
> 
> Overall idea looks correct to me but before merging we should include this parameters in the launch file rather than declaring it in the `teleop.py` file.

hey Peter, is there a way to selectively load parameters from yaml file in ros two? I'm referring [this](https://github.com/ros-planning/moveit2/blob/06e5a0b29107432c8a42e1986d356193383347b8/moveit_ros/moveit_servo/launch/servo_example.launch.py#L39) launch file and it loads all the servo yaml params but we just want to set the `ee_frame_name`. One approach that I was thinking was to create a new yaml from the servo yaml only containing the parameter we want and use that as the launch parameter. What do you think? or are there better options

Reply:
> What do you think? or are there any other better options to approach this?

Hi @abhijelly, thanks for following up on this. If you want to load just one parameter you would need to alter this [function](https://github.com/ros-planning/moveit2/blob/06e5a0b29107432c8a42e1986d356193383347b8/moveit_ros/moveit_servo/launch/servo_example.launch.py#L20). 

It should be possible to return a restricted yaml file.

Reply:
hey @peterdavidfagan - I've passed the `ee_frame_name` as a launch parameter after loading it directly from the servo yaml file. Please let me know if you have any questions/comments/concerns

Reply:
> @abhijelly awesome work. Looks good to me on initial inspection, I will complete a review for this by EOD. Thanks for being persistent on this, good job.

sounds good. Thank you for mentoring this pr!!

Reply:
@peterdavidfagan any updates on this pr?

Reply:
Hey @abhijelly, I am very sorry for the delay on my side on this. I have set aside time today to test this and provide feedback, I'll be online during my evening from about 15:00-19:00.